File: prog02.1.c
#include <stdio.h>

int main() {
    
    int idade, categoria;

    printf("Digite sua idade: ");
    scanf("%d", &idade);

    if (idade < 6) {
        categoria = 1;  
    } else if (idade < 18) {
        categoria = 2;  
    } else if (idade < 60) {
        categoria = 3;  
    } else {
        categoria = 4;  
    }

    switch (categoria) {
        case 1:
            printf("Nao paga passagem (crianca).\n");
            break;
        case 2:
            printf("Paga meia passagem (estudante).\n");
            break;
        case 3:
            printf("Paga passagem inteira.\n");
            break;
        case 4:
            printf("Nao paga passagem (idoso).\n");
            break;
        default:
            printf("Categoria invalida.\n");
            break;
    }

    return 0;
}

File: prog02.1.py
idade = int(input("Digite sua idade: "))

if idade < 6:
    categoria = "crianca"
elif idade < 18:
    categoria = "estudante"
elif idade < 60:
    categoria = "adulto"
else:
    categoria = "idoso"

match categoria:
    case "crianca":
        print("Nao paga passagem (crianca).")
    case "estudante":
        print("Paga meia passagem (estudante).")
    case "adulto":
        print("Paga passagem inteira.")
    case "idoso":
        print("Nao paga passagem (idoso).")
    case _:
        print("Categoria invalida.")